frontend/site-registry/src/app/features/map/MapSearch.graphql

```diff
@@ -1,10 +1,10 @@
 query mapSearch($searchParam: String!) {
-    searchSites(searchParam: $searchParam, page: "1", pageSize: "100") {
+    searchSites(searchParam: $searchParam, page: "1", pageSize: "100000") {
```

Comment:
For now, we're reusing `searchSites` query, but this will change in the PR to follow. I split FE and BE changes for simpler code review.